### PR TITLE
bitfinex - fetchTradingFees

### PR DESCRIPTION
**Pull request by pcriadoperez**
- Add `fetchTradingFee`
- Refactor `fetchTradingFees` to include common format with the fee under all traded symbols

Note: Bitfinex can return different fees depending if crypto, fiat or derivates are traded. Currently there is no function in ccxt that I know of to discern between the type of different symbols. Therefore this PR assumes all symbols have the same trading fee as crypto.

### CODE REVIEW

File: js/bitfinex.js [hunk 1]
```diff
@@ -398,38 +398,119 @@ module.exports = class bitfinex extends Exchange {
         };
     }
 
-    async fetchTradingFees (params = {}) {
+    async fetchTradingFee (symbol, params = {}) {
         await this.loadMarkets ();
+        const safeSymbol = this.safeSymbol (symbol);
```
Comment (kroitor) [inline js/bitfinex.js:403]: We should not use `this.safeSymbol ()` here, instead we use `this.market (symbol)`.

File: js/bitfinex.js [hunk 2]
```diff
@@ -398,38 +398,119 @@ module.exports = class bitfinex extends Exchange {
         };
     }
 
-    async fetchTradingFees (params = {}) {
+    async fetchTradingFee (symbol, params = {}) {
```
Comment (kroitor) [inline js/bitfinex.js:401]: Does this exchange have an endpoint that accepts a market id to return the fee information for just the specific market ? Cause if it does not have it, and we're not sending the market id to the exchange in the request, then we don't need this entire method, cause it's no better than `fetchTradingFees()`. The user can lookup a symbol in the dictionary returned from `fetchFundingFees` in the userland. Hence we don't need this method, if the exchange does not have a corresponding endpoint. Please verify that against their API docs.

File: js/bitfinex.js [hunk 3]
```diff
@@ -398,38 +398,119 @@ module.exports = class bitfinex extends Exchange {
...
+        //          fees_funding_30d: {},
+        //          fees_funding_total_30d: '0',
+        //          fees_trading_30d: {},
+        //          fees_trading_total_30d: '0',
+        //          rebates_trading_30d: {},
+        //          rebates_trading_total_30d: '0',
+        //          maker_fee: '0.001',
+        //          taker_fee: '0.002',
+        //          maker_fee_2crypto: '0.001',
+        //          maker_fee_2stablecoin: '0.001',
```
Comment (frosty00) [inline js/bitfinex.js:484]: i think we should handle the different maker fees here

Comment (pcriadoperez) [reply js/bitfinex.js:484]: I agree. Do we have a way to tell if a symbol is a crypto, stable coins, fiat or derivate?

File: js/bitfinex.js (hunk 2 above)
Comment (pcriadoperez) [reply js/bitfinex.js:401]: I'll remove it from this and other other bitfinex2 PR

**Comment by pcriadoperez** [comment]
This PR is missing a check for stable coins, and applies the default maker and taker fee to stable coins.

I considered using the endpoint `/account_infos` that returns the fee by pairs. However it didn't seem to return derivatives symbols, and also there wasn't any documentation if the LEO discounts needed to be applied to the fees or not. This endpoint also seemed to be deprecated in v2.

**Pull request merged**